solr/core/src/java/org/apache/solr/cli/RunExampleTool.java

```diff
@@ -260,7 +260,7 @@ protected void runExample(CommandLine cli, String exampleName) throws Exception
         "techproducts".equals(exampleName) ? "sample_techproducts_configs" : "_default";
 
     boolean isCloudMode = cli.hasOption('c');
-    String zkHost = cli.getOptionValue('z');
+    String zkHost = SolrCLI.getZkHost(cli);
```

Comment:
Nice one!   I don't think I realized this was here.

Reply:
Apparently this does not work for the specific use case, so I we have to fallback to `SolrCLI#getOptionWithDeprecatedAndDefault`.